en/SUMMARY.md

```diff
@@ -6,9 +6,10 @@
 * [Guide](guide/README.md)
   * [Usage/Paradigms](guide/general_usage.md)
   * [Connecting to Devices](guide/connections.md)
+  * [Managing Devices](guide/using_plugins.md)
   * [Vehicle Information](guide/device_version.md)
   * [Telemetry](guide/telemetry.md)
-  * [Taking Off and Landing](guide/taking_off_landing.md)
+  * [Actions - Take Off, Arm, ...](guide/taking_off_landing.md)
```

Comment:
Renamed to focus on "Actions" old name too specific for broad content of topic.

Reply:
Makes sense.